modules/builder.js

```diff
@@ -507,11 +507,15 @@ function createRelease() {
 			githubApi.config.owner = repoOwner;
 			githubApi.config.token = getEnv('GITHUB_TOKEN');
 
+			const tagName = _this.config.settings.sdkRepo.tagFormat ?
```

Comment:
Use `maybeInit` in the constructor to set a default value of `{version}` so you don't have to have logic here. Other defaults: https://github.com/purecloudlabs/platform-client-sdk-common/blob/master/modules/builder.js#L72

This is just an optional suggestion, but will help reduce validation and conditional setting logic needed in the code.